databases/week2/part-3-more-queries.sql

```diff
@@ -0,0 +1,21 @@
+SELECT task.title, user.name, user.email FROM `task`
+JOIN user_task ON task.id = user_task.task_id
+JOIN user ON user.id = user_task.user_id
+WHERE user.email REGEXP '@spotify.com';
+SELECT user.name, task.title, status.name
+FROM `user`
+JOIN user_task ON user.id = user_task.user_id
+JOIN task ON task.id = user_task.task_id
+JOIN status ON status.id = task.status_id
+WHERE user.name REGEXP 'Donald Duck'
+AND status.name REGEXP 'Not started';
+SELECT user.name, task.title, task.created
+FROM `user`
+JOIN user_task ON user.id = user_task.user_id
+JOIN task ON task.id = user_task.task_id
+JOIN `status` ON status.id = task.status_id
+WHERE user.name REGEXP 'Maryrose Meadows'
+AND month(created) = 09;
+SELECT COUNT(*), monthname(created)
+FROM `task`
+GROUP BY monthname(created);
```

Comment:
Great here, it is always good to include an id (for instance task. id in most of those cases), one can Identify the results easily instead of trying to check with title or description.
example
'```
SELECT task.id,
    task.title AS 'Task Name',
    user.name AS 'user',
    status.name AS status
FROM user_task
    JOIN task ON user_task.task_id = task.id
    JOIN user ON user_task.user_id = user.id
    JOIN status ON task.status_id = status.id
WHERE
    status.name = "Not started"
    AND user.name = 'Donald Duck';
```

Reply:
Thanks for the review. This code looks more readable